committees.yaml

```diff
@@ -125,8 +125,18 @@
   chairs: Naren Dasan
   members:
     - Naren Dasan
+    - Ben Congdon
```

Comment:
Can't comment on line 124, but we should use this PR to update the description as well